ADOPTERS.md

```
@@ -16,6 +16,7 @@ The list includes but is not limited to:
 | Vendor | Spectro Cloud | 2025 | [Link](https://www.spectrocloud.com/) | Spectro Cloud uses OCM in their platform-building platform, Mural. Mural uses OCM for intelligent federation of resources across multiple Kubernetes clusters, simplifying application lifecycle management at scale. |
 | End user | Xiao Hong Shu | 2022 | [Link](xiaohongshu.com) | Xiao Hong Shu (also known as Xiaohongshu or Little Red Book) is a Chinese social media and e-commerce platform. They were very early adopters of OCM and have contributed to the growth of the multi-cluster space. |
 | End user | Guidewire Software Inc. | 2025 | [Link](https://www.guidewire.com/) | Guidewire Software Inc. uses OCM in their Cloud platform for their P&C Insurance customers. To utilize OCM on their platform, they have contributed to the OCM project with implementation of enabling AWS IAM-based registration mechanism on EKS clusters.
+| End user | Groww | 2025 | [Link](https://groww.in/) | Groww uses OCM to deploy application workloads across multiple clusters for cell-based deployment architecture. They built a custom rollout controller on top of OCM that generates ManifestWork in a progressive fashion with manual control from a human operator. |
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)